docs/datatypes/zlayer.md

```diff
@@ -113,6 +130,13 @@ object ZLayerApp0 extends zio.App {
 ```
 
 ### ZLayer example with complex dependencies
+
+ In this example, we can see that `ModuleC` depends upon `ModuleA`, `ModuleB`,
+ and `Clock`. The layer provided to the runnable application shows how
+ dependency layers that are siblings can be combined using `++` into a single
+ combined layer, and then can be provided together as the dependency for another
+ layer.
+
```

Comment:
Minor comment and I realize it was already this way but shouldn't each of these paragraphs be a single line of markdown and then they will be rendered based on the screen versus us manually adding all these line breaks? I feel like this is going to make it look weird and bad on some screens.

Reply:
@adamgfraser do you mean in an editor, or when the markdown is rendered to HTML? FWIW, "weird and bad" is subjective, but I'd like to know what you mean.

It looks perfectly fine in my editor, and markdown renderers will usually ignore line breaks that are not explicitly followed by a space character and combine them into a single `<p>` tag.

I tend to prefer keeping my code to <= 80 chars wide because it makes the code easier to see in a small screen context, or when I have a split-screen view of some code. I don't own the coding styles for ZIO, of course, so I have no real iron in this fire.

I will update my PR to make every paragraph a single line of markdown, let me know if that is what you would prefer.

Reply:
@adamgfraser I've only updated the whitespace of paragraphs that I added as entirely new paragraphs to avoid code churn. I think it's worthwhile to adopt a consistent code formatting convention for markdown if possible. I know little about the options but they exist:
- https://github.com/markdownlint/markdownlint
- https://github.com/DavidAnson/markdownlint

If that seems reasonable, let me know. If we can identify a consensus on formatting markdown I can try to tackle this. Would probably be ideal to unify the formatting of all markdown in a single commit to avoid whitespace commit noise.

Reply:
Maybe it doesn't make that much of a difference. I was worried that manually adding those breaks could for example result in line breaks being inserted too early on wide screens but maybe that is dealt with separately. Looking through the rest of the documentation it looks like one line per paragraph is more common but it is not completely consistent. Anyway, thanks for all your work on this. Will merge when it passes CI.

Reply:
Thanks! I'm all for one-line-per-paragraph. I've seen folks using dedicated markdown editors (as opposed to something IDE-like) and in that context especially that is a much more natural way to format the text.